tests/func/test_state.py

```diff
@@ -78,7 +79,24 @@ def test_remove_unused_links(tmp_dir, dvc):
         result = dvc.state._execute(cmd_count_links).fetchone()[0]
         assert result == 2
 
-        dvc.state.remove_unused_links([])
+        dvc.state.remove_unused_links(["foo", "bar"])
 
         result = dvc.state._execute(cmd_count_links).fetchone()[0]
         assert result == 0
+
+
+def test_get_unused_links(tmp_dir, dvc):
+    assert len(tmp_dir.dvc_gen("foo", "foo_content")) == 1
+    assert len(tmp_dir.dvc_gen("bar", "bar_content")) == 1
```

Comment:
I don't think we have to assert that `dir_helpers` work properly.
Also, we can achieve that using `tmp_dir.dvc_gen({"foo":"foo content", "bar": "bar content"})`

Reply:
I just copy-pasted from above. :stuck_out_tongue: Fixing ...

Reply:
Updated. :slightly_smiling_face: